src/utils/index.ts

```diff
@@ -181,7 +181,7 @@ export const _safewrap = function <F extends (...args: any[]) => any = (...args:
             // @ts-ignore
             return f.apply(this, args)
         } catch (e) {
-            logger.critical('Implementation error. Please turn on debug and contact support@posthog.com.')
+            logger.critical('Implementation error. Please turn on debug mode and open a ticket on https://app.posthog.com/home#panel=support%3Asupport%3A.')
```

Comment:
Could we not change this to a link and just say '[open a support ticket](https://app.posthog.com/home#panel=support%3Asupport%3A)`?

Reply:
AFAIK There is no way you can make an HTML tag interpreted by your the terminal, because your terminal has no ideas what html is. [source](https://dev.to/maxwelladapoe/adding-a-link-to-your-console-log-in-javascript-431o)